diff --git a/tests/wscript b/tests/wscript
index 01fc15a..1f82e26 100644
--- a/src/lib/s4/tests/wscript
+++ b/src/lib/s4/tests/wscript
@@ -27,7 +27,7 @@ def build(bld):
         source = ['runner/memory_status.c'],
         includes = 'utils',
         uselib = 'valgrind',
-        defines = ["HAVE_VALGRIND=%d" % int(bld.env.HAVE_VALGRIND)],
+        defines = ["HAVE_VALGRIND=%d" % int(bld.env.HAVE_VALGRIND or 0)],
         install_path = None,
         )
 
