diff --git a/configure b/configure
index f152cf9..7d6112c 100755
--- a/configure
+++ b/configure
@@ -168,6 +168,10 @@ for opt do
   ;;
   --help|-h) show_help="yes"
   ;;
+  # crosstargeting support is handicap (and autotools support in general
+  # so just giveup and passthru
+  --build=*|--host=*|--target=*|--datadir=*|--sysconfdir=*|--localstatedir=*)
+  ;;
   *) echo "configure: WARNING: unrecognized option $opt"
   ;;
   esac
