--- Makefile.orig	2011-02-25 12:20:21.000000000 +0200
+++ Makefile	2011-02-25 12:53:00.902879997 +0200
@@ -18,11 +18,6 @@
 
 # FLAGS ####################################################
 
-ifneq ($(MAKECMDGOALS), release)
-    EXTRA_CFLAGS += -g3 -O0 -Werror
-else
-    EXTRA_CFLAGS += -g0 -O2 -DNDEBUG
-endif
 EXTRA_CFLAGS += -MMD -pedantic -Wall -Wextra -Wno-missing-field-initializers
 
 HG_REV := $(shell hg id -ib && touch src/hg_revision.h)
