Index: configure.in
===================================================================
--- configure.in	(revision 11795)
+++ configure.in	(working copy)
@@ -225,7 +225,7 @@
         kernel=`uname -r`
 
         case "${kernel}" in
-             2.6.*) 
+             2.6.*|3.*) 
         	    AC_MSG_RESULT([2.6 family (${kernel})])
         	    AC_DEFINE([KERNEL_2_6], 1, [Define to 1 if you're using Linux 2.6.x])
         	    ;;
